src/controllers/dynakube/version/updater.go

```diff
@@ -180,15 +189,24 @@ func updateVersionStatusForTenantRegistry( //nolint:revive
 }
 
 func getTagFromImageID(imageID string) (string, error) {
-	ref, err := reference.Parse(imageID)
+	ref, err := name.ParseReference(imageID)
 	if err != nil {
 		return "", err
 	}
-	taggedRef, ok := ref.(reference.NamedTagged)
-	if !ok {
-		return "", errors.New("no tag found to check for downgrade")
+
+	var taggedRef name.Tag
```

Comment:
I think you can remove this declaration since we're not using it anymore in return.